doc/sphinx/source/conf.py

```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.0.1'
+version = '1.12'
 # The full version, including alpha/beta/rc tags.
-release = '0.0.1'
+release = '1.12.0'
```

Comment:
They aren't being used anywhere right now, but we can start including `|release|` and `|version|` in reST docs and it will be replaced with `1.12.0` and `1.12`, respectively.